File: src/util/NetUtil.java
package util;

import java.net.InetAddress;

/**
 * Created by SAM on 2018/5/10.
 */
public class NetUtil {
    public static String  getHostAddress(){
        String ip = "未知";
        try {
            InetAddress  address = InetAddress.getLocalHost();
            ip = address.getHostAddress();
        } catch(Exception e) {
            System.out.println(e);
        }
        return  ip;
    }
    public static String  getHostName(){
        String hostname = "未知";
        try {
            InetAddress  address = InetAddress.getLocalHost();
            hostname = address.getHostName();
        } catch(Exception e) {
            System.out.println(e);
        }
        return  hostname;
    }

}

File: src/util/NetUtil.scala
/*package util

/**
  * 网络工具类
  */
object NetUtil {
  def getHostAddress(): String = {
    var ip = "未知"
    try {
      import java.net.InetAddress
      val address = InetAddress.getLocalHost
      ip = address.getHostAddress
    } catch {
      case e: Exception => {
        e.printStackTrace()
      }
    }
    ip
  }

  def getHostName(): String = {
    var hostname = "未知"
    try {
      import java.net.InetAddress
      val address = InetAddress.getLocalHost
      hostname = address.getHostName
    } catch {
      case e: Exception => {
        e.printStackTrace()
      }
    }
    hostname
  }
}*/
